### ISSUE TITLE
What was the first project you created as a beginner?

### ISSUE THREAD

**Issue by vadimdemedes** [Qix-/ama#10]

**Comment by Qix-** [comment]
This is a hard one to answer as I've been writing code since well before 9 years old (I was nine when I bought the compiler for [Liberty Basic](http://www.libertybasic.com/)). In fact, up until a few years ago I was able to still find remnants of my first public posts on the internet, ever. However, I remember learning HTML from a small guide my cousin found on Ask Jeeves when I was 5 or 6.

The first project I'd really consider anything substantial was probably a PHP framework called Sinimin (inspired by my then-alias "SiNT4X") that I still have laying around on a CD somewhere. I don't remember why it was "so good" but I remember slaving away at SMS gateways and weird magic method hacking that would be considered poor form nowadays. I was a freshman in high school and the project was encouraged by my intro to tech teacher Lee Eyerman. Think wordpress, but by a single 16 year old.

Prior to that, I was just learning stuff and playing around with languages and APIs, talking with other skilled developers on hacking forums and whatnot. I remember writing hacks for a game called Combat Arms when I was about 13/14 years old working with a guy named Rajinn (sp?) writing remote authentication and connectivity/control classes in C++ using the IRC protocol. That was fun. I learned socket programming at a relatively young age - probably 12 or 13 - when I tried to use [AutoIt](https://www.autoitscript.com/site/autoit/) in middle school to do probably not-so-kosher things and ended up getting my entire file (including homework) deleted and my account banned for a few weeks.

Good question :)

/longanswer
